static/hub.ts

```diff
@@ -358,7 +361,7 @@ export class Hub {
         this.editors.push(editor);
     }
 
-    private treeFactory(container: GoldenLayout.Container, state: any): Tree {
+    private treeFactory(container: GoldenLayout.Container, state: ConstructorParameters<typeof Tree>[1]): Tree {
```

Comment:
This is "Get me the type of the n-th parameter of this type's constructor". Note that Tree here gets the index 1 while the rest get the second as the order for state is switched for the Tree view.

Reply:
Whoah 🤯